frontend/views/components/Avatar.vue

```diff
@@ -31,7 +31,7 @@ export default ({
   },
   mounted () {
     console.log(`Avatar under ${this.$parent.$vnode.tag} blobURL:`, this.blobURL, 'src:', this.src)
-    if (typeof this.src === 'object') {
+    if (this.src && typeof this.src === 'object') {
```

Comment:
Why was this change made?

Reply:
This fixes a defect in the previous files PR, since `typeof null === 'object'`.

Reply:
Great catch!! Could you add a single comment for this above line 34? I feel like a lot of people would get tripped up and confused by this JavaScript quirk.